lib/util/expected-state.js

```diff
@@ -174,6 +174,13 @@ module.exports = ExpectedState = {
       );
     }
 
+    // TODO
```

Comment:
Is this `TODO` still valid?

Reply:
that TODO is valid, but it's referencing #213 . Probably worth removing now and adding in when that issue is tackled.